### ISSUE TITLE
NVDA cannot find the mouse pointer if it is moved with a script

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4538]
**Reported by andrewd on 2014-10-09 05:26**
Some software can be made more accessible by moving the mouse pointer to a specific point with - say - a Autohotkey script.  Unlike when the physical mouse is moved, NVDA is not aware of the move and setting the navigation object to the object under the mouse does not work unless the mouse is clicked.  Clicking with the script suffices, but sometimes a click is not desirable.  Is it possible to have NVDA recognise the mouse pointer position regardless of whether it is moved via the physical mouse or with a script?

**Comment by bhavyashah** [comment]
@jcsteh @michaelDCurran Since this ticket relates to NVDA's support for a mouse, which might possibly require the intervention of a sighted developer, I would like to merely confirm if sufficient information to assess this issue has been gathered and analysed, or if further diagnostic tests are required, for which @feerrenrut and @Qchristensen may be ideal.

**Comment by feerrenrut** [comment]
I think the main thing missing from this is some concrete example. When trying to implement this the developer will need to come up with a method of testing it. We need a test case that demonstrates how moving the mouse with a script results in a better accessibility outcome. Also handy would be a set of steps to for the implementer to have some certainty that they will avoid regressions. Such as causing a loop by triggering NVDA to move the mouse.

**Comment by Adriani90** [comment]
@leonardder now that injected mouse events are supported, is this issue solved?

**Comment by leonardder** [comment]
I guess so. If the problem still persists in a particular situation, it would be best to create a new issue for it.
